packages/cli-repl/src/run.ts

```diff
@@ -1,4 +1,4 @@
-import { CliRepl, parseCliArgs, mapCliToDriver, getStoragePaths, USAGE } from './index';
+import { CliRepl, parseCliArgs, mapCliToDriver, getStoragePaths, getMongocryptdPath, USAGE } from './index';
```

Comment:
what is the reason for importing these though `./index` and not directly? Is there something we get in return to exposing  `CliRepl, parseCliArgs, mapCliToDriver, getStoragePaths, getMongocryptdPath, USAGE` ... externally?

Reply:
No, but I don’t think we’re considering any of this public API to begin with, right? Or at least not at this point? There’s no particular reason for doing it this way other than that that’s what we’ve been doing for the other things listed here as well